src/framework/components/element/system.js

```diff
@@ -45,6 +48,28 @@ pc.extend(pc, function () {
         this.defaultImageMaterial.depthWrite = false;
         this.defaultImageMaterial.update();
 
+        this.defaultImageMaskMaterial = this.defaultImageMaterial.clone();
+        this.defaultImageMaskMaterial.diffuse = new pc.Color(0,0,0,1); // black diffuse color to prevent ambient light being included
+        this.defaultImageMaskMaterial.emissive = new pc.Color(0.5,0.5,0.5,1); // use non-white to compile shader correctly
```

Comment:
You can just use .set to set this properties. Otherwise you're just 'leaking' a pc.Color object. Plus you only need to set 3 values. Diffuse and emissive properties are RGB, not RGBA.